ocaml/idl/datamodel_vtpm.ml

```diff
@@ -77,22 +77,26 @@ let t =
          ; allowed_and_current_operations operations
          ; [
              field ~qualifier:StaticRO ~ty:(Ref _vm) "VM"
+               ~in_product_since:rel_rio
                "The virtual machine the TPM is attached to"
            ; field ~qualifier:DynamicRO ~ty:(Ref _vm) "backend"
-               ~default_value:(Some (VRef null_ref))
+               ~in_product_since:rel_rio ~default_value:(Some (VRef null_ref))
                "The domain where the backend is located (unused)"
            ; field ~qualifier:DynamicRO ~ty:persistence_backend
-               ~default_value:(Some (VEnum "xapi")) ~lifecycle:[]
-               "persistence_backend" "The backend where the vTPM is persisted"
-           ; field ~qualifier:StaticRO ~ty:Bool
+               ~in_product_since:rel_rio ~default_value:(Some (VEnum "xapi"))
```

Comment:
This isn't rel_rio though, it is 22.26.0, see datamodel_lifecycle.ml.

Reply:
And that is ignored `~in_product_since` here is ignored, as `~lifecycle` has been specified. A better way forward would be to eliminate `~in_product_since` and use `~lifecycle` everywhere.